CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# iwant-bot
+---
+## Hello everyone!
+- :star2: If you are reading this document then I'm assuming that you are interested in contributing to the __iwant-bot__ project.
+- This project is made by Kiwi.com interns. 
+- All contributions are welcome: new use-cases, code, patches, bugfixes, issues, etc. 
+- This is just set of guidelines, not rules. Use your best judgement, and feel free to propose changes in a pull request.
+
+
+## Ways to contribute
+---
+  - At first, check existing [issues](https://github.com/kiwicom/iwant-bot/issues) and our pull requests and help us with development with your proposed solution or improvement

```

Comment:
"and help us with development" sounds bad somehow, dunno how to mkae it sound better tho